src/Farmer/Builders/Builders.ContainerApps.fs

```diff
@@ -34,7 +34,12 @@ type ContainerAppConfig =
       ImageRegistryCredentials : ImageRegistryAuthentication list
       Containers : ContainerConfig list
       Dependencies : Set<ResourceId> }
-
+    member this.ResourceId = containerApps.resourceId this.Name
```

Comment:
I don't think [line 51](https://github.com/CompositionalIT/farmer/pull/922/files#diff-b82693f74fcd946fa59b9816bacfd8d576ea0920bbd9a9da3bb293d29044cac3R51) is correct actually, wondering if I should fix that too. I'm thinking it should be `managedEnvironments.resourceId`.